### DISCUSSION TITLE
Run PaddleOCR in Azure

### DISCUSSION THREAD

**Discussion by francisjump** [PaddlePaddle/PaddleOCR#13751] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

My error message is shown below:
C++ Traceback (most recent call last):

0 paddle_infer::Predictor::Predictor(paddle::AnalysisConfig const&)

1 std::unique_ptr<paddle::PaddlePredictor, std::default_deletepaddle::PaddlePredictor > paddle::CreatePaddlePredictor<paddle::AnalysisConfig, (paddle::PaddleEngineKind)2>(paddle::AnalysisConfig const&)

2 paddle::AnalysisPredictor::Init(std::shared_ptrpaddle::framework::Scope const&, std::shared_ptrpaddle::framework::ProgramDesc const&)

3 paddle::AnalysisPredictor::PrepareProgram(std::shared_ptrpaddle::framework::ProgramDesc const&)

4 paddle::AnalysisPredictor::OptimizeInferenceProgram()

5 paddle::inference::analysis::Analyzer::RunAnalysis(paddle::inference::analysis::Argument*)

6 paddle::inference::analysis::IrAnalysisPass::RunImpl(paddle::inference::analysis::Argument*)

7 paddle::inference::analysis::IRPassManager::Apply(std::unique_ptr<paddle::framework::ir::Graph, std::default_deletepaddle::framework::ir::Graph >)

8 paddle::framework::ir::Pass::Apply(paddle::framework::ir::Graph*) const

9 paddle::framework::ir::SelfAttentionFusePass::ApplyImpl(paddle::framework::ir::Graph*) const

10 paddle::framework::ir::GraphPatternDetector::operator()(paddle::framework::ir::Graph*, std::function<void (std::map<paddle::framework::ir::PDNode*, paddle::framework::ir::Node*, paddle::framework::ir::GraphPatternDetector::PDNodeCompare, std::allocator<std::pair<paddle::framework::ir::PDNode* const, paddle::framework::ir::Node*> > > const&, paddle::framework::ir::Graph*)>)

Error Message Summary:

FatalError: Illegal instruction is detected by the operating system.

[TimeInfo: *** Aborted at 1722584852 (unix time) try "date -d @1722584852" if you are using GNU date ***]

[SignalInfo: *** SIGILL (@0x7f3b9325a82a) received by PID 7796 (TID 0x7f3b9a2c5080) from PID 18446744071883302954 ***]

As suggested by the admin from Azure, he asks me to find the admin here to solve the problem.  I would like to run the paddleocr in the Azure cloud platform, and I cannot find the similar solution. Please give me some suggestion on this issue . Thank you.

### 🏃‍♂️ Environment (运行环境)

The setting of VM is shown below:
Operating system: Linux (ubuntu 24.04)
Size: Standard B8ms (8 vcpus, 32 GiB memory)

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

ocr = PaddleOCR(use_angle_cls=True, lang="ch")

**Comment by SWHL** [comment]
Please confirm if the paddlepaddle has been successfully installed first by the [docs](https://www.paddlepaddle.org.cn/)

**Comment by francisjump** [comment]
which one should I install based on this operating system?

**Reply by SWHL** [reply]
Install according to the documentation

**Comment by francisjump** [comment]
I face another error
C++ Traceback (most recent call last):
--------------------------------------
0   paddle_infer::Predictor::Predictor(paddle::AnalysisConfig const&)
1   std::unique_ptr<paddle::PaddlePredictor, std::default_delete<paddle::PaddlePredictor> > paddle::CreatePaddlePredictor<paddle::AnalysisConfig, (paddle::PaddleEngineKind)2>(paddle::AnalysisConfig const&)
2   paddle::AnalysisPredictor::Init(std::shared_ptr<paddle::framework::Scope> const&, std::shared_ptr<paddle::framework::ProgramDesc> const&)
3   paddle::AnalysisPredictor::PrepareProgram(std::shared_ptr<paddle::framework::ProgramDesc> const&)
4   paddle::AnalysisPredictor::OptimizeInferenceProgram()
5   paddle::inference::analysis::Analyzer::RunAnalysis(paddle::inference::analysis::Argument*)
6   paddle::inference::analysis::IrAnalysisPass::RunImpl(paddle::inference::analysis::Argument*)
7   paddle::inference::analysis::IRPassManager::Apply(std::unique_ptr<paddle::framework::ir::Graph, std::default_delete<paddle::framework::ir::Graph> >)
8   paddle::framework::ir::Pass::Apply(paddle::framework::ir::Graph*) const
9   paddle::framework::ir::SelfAttentionFusePass::ApplyImpl(paddle::framework::ir::Graph*) const
10  paddle::framework::ir::GraphPatternDetector::operator()(paddle::framework::ir::Graph*, std::function<void (std::map<paddle::framework::ir::PDNode*, paddle::framework::ir::Node*, paddle::framework::ir::GraphPatternDetector::PDNodeCompare, std::allocator<std::pair<paddle::framework::ir::PDNode* const, paddle::framework::ir::Node*> > > const&, paddle::framework::ir::Graph*)>)

----------------------
Error Message Summary:
----------------------
FatalError: `Illegal instruction` is detected by the operating system.
  [TimeInfo: *** Aborted at 1728377171 (unix time) try "date -d @1728377171" if you are using GNU date ***]
  [SignalInfo: *** SIGILL (@0x7a5dfaa8860a) received by PID 12269 (TID 0x7a5dce4006c0) from PID 18446744073619932682 ***]

**Reply by SWHL** [reply]
Please check whether the paddlepaddle framework is successfully installed according to the [docs](https://www.paddlepaddle.org.cn/install/quick?docurl=/documentation/docs/zh/install/pip/linux-pip.html)

**Comment by UserWangZz** [comment]
This may be a problem with your environment.

**Comment by Daveboot** [comment]
when only install the package paddlepaddle-gpu, code is correct, but use paddlepaddle packages, the same error as above occurred. Can i fixed it?
